README.md

```diff
@@ -833,22 +868,23 @@ client.mutateGraphWith(mutation) { result, error in
     guard error == nil else {
         // handle request error
     }
-    
+
     guard let userError = result?.checkoutCreate?.userErrors else {
         // handle any user error
         return
     }
-    
+
     let checkoutID = result?.checkoutCreate?.checkout?.id
 }
 ```
-**It is best practice to always include `userErrors` fields in your mutation payload query, where possible.** You should always validate user input before making mutation requests but it's possible that there might be a mismatch between the client and server. In this case, `userErrors` will contain an error with a `field` and `message` for any invalid or missing fields.
+
+**It is best practice to always include `userErrors` fields in your mutation payload query, where possible.** You should always validate user input before making mutation requests, but it's possible that a validated user input might cause a mismatch between the client and server. In this case, `userErrors` contains an error with a `field` and `message` for any invalid or missing fields.
 
 Since we'll need to update the checkout with additional information later, all we need from a checkout in this mutation is an `id` so we can keep a reference to it. We can skip all other fields on `Storefront.Checkout` for efficiency and reduced bandwidth.
 
 #### Updating a checkout [⤴](#table-of-contents)
 
-A customer's information may not be available when a checkout is created. The Buy SDK provides mutations for updating specific checkout fields that are required for completion. Namely the `email` and `shippingAddress` fields:
+A customer's information may not be available when a checkout is created. The Buy SDK provides mutations for updating specific checkout fields that are required for completion, such as the `email` and `shippingAddress` fields:
```

Comment:
are `email` and `shippingAddress` just two of a few different fields that are required, or are they the only two that are required?